### PR TITLE
Add `setSoaEdit` and `setSoaEditApi` methods to Zone class

### PR DESCRIPTION
**Pull request by mvdgeijn**
## Description

Added the setSoaEdit and setSoaEditApi methods to the Zone class to be able to set the soa_edit and soa_edit_api domain metadata.

## Motivation and context

Doesn't fix an open issue, just adding an extra feature

## How has this been tested?

New methods tested on a live PowerDNS server v4.6

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our [continuous integration](http://www.phptherightway.com/#continuous-integration) server to make sure your [tests pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](../.github/CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] My pull request contains a title that can be used as a release note.
- [x] I have added the appropriate labels to my pull request (e.g. breaking-change, new-feature, bugfix).
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

If you're unsure about any of these, don't hesitate to ask. We're here to help!

### CODE REVIEW

**Comment by mvdgeijn** [comment]
How can I add a label to this PR? The label should be 'new-feature'

File: src/Zone.php [hunk 1]
```diff
@@ -248,4 +250,28 @@ public function dnssec(): Cryptokey
...
+     * Set a new value for the SOA_EDIT_API setting for this zone
+     *
+     * @param string $value New value for the soa_edit_api meta setting
+     *
+     * @return bool True when the request succeeded.
+     */
+    public function setSoaEditApi(string $value): bool
+    {
+        return $this->put(new SoaEditApiTransformer(['soa_edit_api' => $value]));
+    }
```
Comment (robbinjanssen) [inline src/Zone.php:276]:
```suggestion
    /**
     * Set a new value for the SOA_EDIT setting for this zone.
     *
     * @param string $value New value for the soa_edit meta setting.
     *
     * @return bool True when the request succeeded.
     */
    public function setSoaEdit(string $value): bool
    {
        return $this->put(new SoaEditTransformer(['soa_edit' => $value]));
    }

    /**
     * Set a new value for the SOA_EDIT_API setting for this zone.
     *
     * @param string $value New value for the soa_edit_api meta setting.
     *
     * @return bool True when the request succeeded.
     */
    public function setSoaEditApi(string $value): bool
    {
        return $this->put(new SoaEditApiTransformer(['soa_edit_api' => $value]));
    }
```

**Review by robbinjanssen — CHANGES_REQUESTED** [review]
Added a few dots at the end of lines :-)

**Comment by robbinjanssen** [comment]
@mvdgeijn tagged as [v4.1.0](https://github.com/exonet/powerdns-php/releases/tag/v4.1.0). Thanks for your contribution!

**Pull request merged**